content/docs/reference/install.md

```diff
@@ -224,9 +224,9 @@ warning: A new version of Pulumi is available. To upgrade from version '0.17.26'
 or visit https://pulumi.com/docs/reference/install/ for manual instructions and release notes.
 ```
 
-<div class="note note-warning" role="alert">
+<div class="note note-info" role="alert">
     <p>
-        <span>Skip version check.</span> If you're in an environment with no internet access, you may skip the Pulumi version update check by setting the environment variable <code>PULUMI_SKIP_UPDATE_CHECK</code> to <code>1</code> or <code>true</code>.
+        <span>Skip version check:</span> If you're in an environment with no internet access, you may skip the Pulumi version update check by setting the environment variable <code>PULUMI_SKIP_UPDATE_CHECK</code> to <code>1</code> or <code>true</code>.
```

Comment:
I see others on this page are capped, but this one isn't. We should probably choose one or the other (I'm generally in favor of not-caps as a rule, excepting things like headings and labels.)